src/main/java/seedu/address/scraper/ScraperManager.java

```diff
@@ -432,7 +431,9 @@ public void getUngradedQuests(WebElement filterBar) {
     private void saveToStorage(List<Mission> missions, List<Quest> quests, List<Student> students, String name)
             throws IOException {
         try {
-            model.setName(name);
+            if (!name.isEmpty()) {
```

Comment:
Awesome find, thank you for fixing the bug!

Reply:
how u leave a review on ur phone

Reply:
or u back in ur room alr